mipengine/controller/api/algorithm_specifications_dtos.py

```diff
@@ -34,6 +34,8 @@ class ParameterType(str, Enum):
     INT = "int"
     TEXT = "text"
     BOOLEAN = "boolean"
+    ENUM_FROM_LIST = "enum_from_list"
+    ENUM_FROM_CDE = "enum_from_cde"
```

Comment:
Didn't we say to add these in a separate field? Also if it's `enum_from_list` what list are we talking about, shouldn't we provide it?

Reply:
It is stated in the commit message that this is a temporary solution, in order to move forward with the implementation of Logistic. This feature will generate changes in various places (validator, test case generator, backend API?, ...) and would drag this PR for a long time. I created a ticket which is also linked in the commit message.

The list you are talking about is basically the `enums` field in the algorithm parameter, so this case is already covered. The problem was with `enum_from_cde` where the enums are not known at writing time because they come from the particular variable that is selected by the user for the current experiment.